NachoSpigot-API/pom.xml

```diff
@@ -10,29 +10,19 @@
     </parent>
 
     <artifactId>api</artifactId>
-    <version>1.8.8-R0.2-SNAPSHOT</version>
+    <version>1.8.8-R0.3-SNAPSHOT</version>
     <packaging>jar</packaging>
 
     <name>NachoSpigot-API</name>
     <description>An enhanced plugin API for Minecraft servers.</description>
 
     <properties>
-        <!--PaperSpigot - Bump to 1.8 - This will haunt me -->
         <additionalparam>-Xdoclint:none</additionalparam>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>15</maven.compiler.source>
+        <maven.compiler.target>15</maven.compiler.target>
```

Comment:
https://github.com/CobbleSword/NachoSpigot/pull/159#discussion_r707199048

Reply:
You only changed the server back to Java 8 and now the server won't compile :p

Reply:
Gadzooks! I'll fix that now haha, I must've missed it

Reply:
Instead of dropping java 8 without any reason, do a issue about why nachospigot should only support java 15-17

Reply:
It was on accident and already corrected.